osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/configuration/service/AttributeAddressFactory.java

```diff
@@ -0,0 +1,19 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.configuration.service;
+
+import org.openmuc.jdlms.AttributeAddress;
+import org.openmuc.jdlms.ObisCode;
+
+public class AttributeAddressFactory {
```

Comment:
this factory returns always one and the same object. I assume this class will be extended in the near future? if not, it seems overkill to create a factory for just a single object.

Reply:
Not overkill, encapsulation.